web/app/frontend/__tests__/utils/storage.test.js

```diff
@@ -1,17 +1,73 @@
-import {storeEncryptedLocalData} from '../../src/utils/storage';
-import AsyncStorage from '@react-native-community/async-storage';
-import * as SecureStore from "expo-secure-store";
-import * as CryptoJS from "react-native-crypto-js";
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import * as SecureStore from 'expo-secure-store';
+import CryptoJS from 'react-native-crypto-js';
 
-jest.mock('@react-native-community/async-storage');
-jest.mock('expo-secure-store');
-jest.mock('react-native-crypto-js');
+import {
+    deleteEncryptedLocalData,
+    getEncryptedLocalData,
+    storeEncryptedLocalData,
+} from '../../src/utils/storage';
 
-describe("utils/storage", () => {
-    it("should encrypt data and store it", () => {
+const recordId = 'mock';
+const key = 'whatever';
+const data = { something: 'ok' };
+SecureStore.getItemAsync.mockResolvedValue(key);
+CryptoJS.AES.decrypt.mockResolvedValue(JSON.stringify(data));
+let encryptedData;
+(async () => {
+    encryptedData = (
+        await CryptoJS.AES.encrypt(JSON.stringify(data), key)
+    ).toString();
+    CryptoJS.AES.encrypt.mockResolvedValue(encryptedData);
+    AsyncStorage.getItem.mockResolvedValue(encryptedData);
+})();
```

Comment:
this stuff is usually done within the describe, I think it would be cleaner that way. 
then you also wouldn't have to do this closure to prepare the encryptedData